app/assets/stylesheets/components/beta_tag/_beta_tag.scss

```diff
@@ -0,0 +1,11 @@
+.beta-tag {
+  @include inline-block();
+  @include body(18, 24);
+  font-weight: 700;
+  position: relative;
+  bottom: -$baseline-unit*5;
```

Comment:
I'm all for single-direction margin (zeroing out `margin-top`) however that would mean overriding all of the styles coming from Yeast, both base styles and the class names and then using project-based typography.